lib/rules/named-grid-areas-no-invalid/__tests__/index.js

```diff
@@ -12,14 +13,21 @@ testRule({
 			code: 'a { grid-template-areas: "a a a" "b b b"; }',
 		},
 		{
-			code: `div { grid-template-areas: "head head"
-"nav  main"
-"nav  foot";}`,
+			code: stripIndent`
```

Comment:
is this correct? let me know. https://github.com/stylelint/stylelint/pull/5167#discussion_r580942640